Plugins/NativeCamera/Source/Apple/CameraDevice.mm

```diff
@@ -20,10 +20,24 @@
 
 @class CameraTextureDelegate;
 
-@interface CameraTextureDelegate : NSObject <AVCaptureVideoDataOutputSampleBufferDelegate>
+struct VideoRotationAngle
 {
-    @public AVCaptureVideoOrientation VideoOrientation;
+    enum Enum
+    {
+        LandscapeRight,
+        LandscapeLeft,
+        Portrait,
+        PortraitUpsideDown
+    };
+};
 
+@interface CameraTextureDelegate : NSObject <AVCaptureVideoDataOutputSampleBufferDelegate, NSObject>
```

Comment:
Why is the extra `, NSObject` needed? Isn't that supposed to be a list of protocols / interfaces being implemented?

Reply:
leftover. removed.